Tweaks to internal organ damage from trauma

comma opened the pull request:
Chance scales up with damage now.
If organ is encased (ribs, skull etc), and that's not broken, chance is halved.

Removes bullets doing organ damage TWICE, first in bullet_act and then in take_damage. Bullet act one made it pretty much guaranteed to deal full bulletworth damage to an organ.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

PurpleMartinJCK reviewed (approved):
Looks good to me. I think this system will work better.

comma committed:
Tweaks to internal organ damage from trauma

Chance scales up with damage now.
If organ is encased (ribs, skull etc), and that's not broken, chance is halved.

Removes bullets doing organ damage TWICE, first in bullet_act and then in take_damage. Bullet act one made it pretty much guaranteed to deal full bulletworth damage to an organ.
